hi-ml/src/health_ml/run_ml.py

```diff
@@ -253,8 +253,9 @@ def run_training(self) -> None:
             assert self.trainer, "Trainer should be initialized before training. Call self.init_training() first."
             self.trainer.fit(self.container.model, datamodule=self.data_module)
 
-        assert self.trainer.logger is not None
-        self.trainer.logger.finalize('success')
```

Comment:
is logger deprecated now?

Reply:
It will be in 1.8. This change is to prevent it from logging a warning here (about the deprecation)